Automattic/jetpack
# Fix linting issues in tiled gallery

#### Changes proposed in this Pull Request:

* This PR fixes some linting issues in the `tiled-gallery` block, including previous deprecations, and removes the relevant files from the eslint exclude list
* I also fixed some typos in the documentation and comments

#### Other information:


#### Jetpack product discussion

No product changes -- just better linting.

#### Does this pull request change what data or activity we track or use?

No changes to activity tracking or data.

#### Testing instructions:

* Inspect the changes and/or check the linter output from the automated tests
* I don't think there are any specific actions we need to take beyond that
